### DISCUSSION TITLE
More Selectively Load Shows Throughout Medusa App

### DISCUSSION THREAD

**Discussion by trumblejoe** [pymedusa/Medusa#9091] [Ideas & Feature requests]
Is there a need/enhancement opportunity to be more selective with where the show database is loading all shows? I was searching/snatching a single TV show season and I observed that all 169 of my TV Shows were being loaded in the background per the green status bar, that seems unnecessary on anything except mass overview pages like the Home page.

**Comment by medariox** [comment]
The bar does not reflect which shows are being loaded.
